test/common/test_models.py

```diff
@@ -92,3 +92,10 @@ def test_get_bitmap_angle(mock_bitmap):
 def test_estimate_orientation(mock_image):
     angle = estimate_orientation(mock_image)
     assert abs(angle - 30.) < 1.
+
+
+def test_rotate_page(mock_image):
```

Comment:
I would argue it's actually `test_estimate_orientation` :)

Reply:
Indeed ! I am being a scatterbrain !